cmd/create/operatorroles/cmd.go

```diff
@@ -162,7 +162,9 @@ func run(cmd *cobra.Command, argv []string) {
 	r := rosa.NewRuntime().WithAWS().WithOCM()
 	defer r.Cleanup()
 
-	rosa.HostedClusterOnlyFlag(r, cmd, vpcEndpointRoleArnFlag)
+	if !cmd.Flag("hosted-cp").Changed {
```

Comment:
Not entirely sure if I follow, also considering the cmd is suplied to the function this could be done internally instead :thinking:

Reply:
Updated func